operator/api/v1alpha1/elastiservice_types.go

```diff
@@ -30,13 +30,12 @@ const (
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 // ElastiServiceSpec defines the desired state of ElastiService
+// +kubebuilder:validation:Required={"scaleTargetRef","service"}
 type ElastiServiceSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
-	// +kubebuilder:validation:Required
```

Comment:
These old fields weren't effective, even though it is mentioned in the Docs, but didn't have any effects. 
Have validated it now.

Reply:
As discussed, we will move to different validation for CRDs in future.